bin/p2-exec/main.go

```diff
@@ -39,8 +39,8 @@ func main() {
 		os.Clearenv()
 	}
 
-	if *envDir != "" {
-		err := loadEnvDir(*envDir)
+	for _, dir := range *envDir {
```

Comment:
Small request: It'd be nice if the arg description said in what order the env directories are processed - thinking about situations where a launchable needs to override a global value or something.